lib/test_server.py

```diff
@@ -4,21 +4,25 @@
 from datetime import datetime
 
 def setup_module():
-  group = Group(
+  Group(
       name="Test",
       description="Test",
       access_code="test",
       rules="test",
       created=datetime.utcnow
   ).save()
-  group = Group(
-      name="Test",
-      description="Test",
+
+  Group(
+      name="Test2",
+      description="Test2",
       access_code="test2",
-      rules="test",
+      rules="test2",
       created=datetime.utcnow
   ).save()
 
+def teardown_function():
+  Connection.drop_collection()
+
 def teardown_module():
```

Comment:
We can probably get rid of the Connection drop in the module tear down and just do it in the function